### PR TITLE
[feature]: support mmdet models config

### PR DESCRIPTION
**Pull request by Cathy0908**
Usage guide please refer to `docs/source/tutorials/mmlab_models_usage_guide.md`

### CODE REVIEW

**Review by wenmengzhou — COMMENTED** [review]
tutorials for using mmdet models should also be written

File: easycv/utils/config_tools.py [hunk 1]
```diff
@@ -17,6 +15,14 @@
     import re
 
 
+def pre_check_config(cfg):
```
[the remark this answers is not in the document]
Comment (Cathy0908) [reply easycv/utils/config_tools.py]: done!

File: tools/train.py [hunk 2]
```diff
@@ -146,6 +147,8 @@ def main():
     if args.load_from is not None:
         cfg.load_from = args.load_from
 
+    pre_check_config(cfg)
```
[the remark this answers is not in the document]
Comment (Cathy0908) [reply tools/train.py]: done!

File: docs/source/tutorials/mmlab_models_usage_guide.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Use mmlab's models in EasyCV
```
Comment (wenmengzhou) [inline docs/source/tutorials/mmlab_models_usage_guide.md:1]: mmdetection?

File: docs/source/tutorials/mmlab_models_usage_guide.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+]
+```
+
+> Parameters:
+>
+> 	- type: the name of the open source, only `mmdet` is supported
+> 	- name:  the name of api
+> 	- Module: The name of the module to which the api belongs, only  `backbone`,`head`,`neck`,`model` are supported.
+
+In this configuration , the `head`, `neck`, and `model` parts specify the type as `mmdet`, except for `backbone`.
```
Comment (wenmengzhou) [inline docs/source/tutorials/mmlab_models_usage_guide.md:45]: for other models in mmdetection, what modules should user declare?  we should a give a brief description, and provide more config examples in the future

File: tools/eval.py [hunk 5]
```diff
@@ -143,6 +144,9 @@ def main():
     if cfg.get('oss_io_config', None) is not None:
         io.access_oss(**cfg.oss_io_config)
 
+    # dynamic adapt mmdet models
```
Comment (wenmengzhou) [inline tools/eval.py:147]: this file has related issue: #53   

we may remove this file, but it seems pose evaluation is supported in this file

**Review by wenmengzhou — COMMENTED** [review]
add new comments

File: docs/source/tutorials/mmlab_models_usage_guide.md (hunk 4 above)
Comment (Cathy0908) [reply docs/source/tutorials/mmlab_models_usage_guide.md:45]: For details, you can refer to the doc and case in line9,lin61,line65

File: tools/eval.py (hunk 5 above)
Comment (Cathy0908) [reply tools/eval.py:147]: yes, pose is  supported,  but it‘s ok to just keep one

File: docs/source/tutorials/mmlab_models_usage_guide.md (hunk 3 above)
Comment (Cathy0908) [reply docs/source/tutorials/mmlab_models_usage_guide.md]: done!

**Review by wenmengzhou — APPROVED** [review]

**Pull request merged**